tools/cellprofiler/common.xml

```diff
@@ -0,0 +1,444 @@
+<tool id="cp-common" name="Common" version="@CP_VERSION@">

```

Comment:
Maybe replace here Common with "Cellprofiler common start modules" or something? Look in your tool panel, currently, this looks wired left?